fast/stages/3-network-security/data/firewall-policy-rules/dev/egress.yaml

```diff
@@ -20,3 +20,4 @@ egress-inspect-internet:
       - "0.0.0.0/0"
   action: "apply_security_profile_group"
   security_profile_group: "dev"
+  # tls_inspect: true
```

Comment:
If this being commented out is intended, please add a comment explaining what it does, otherwise uncomment.

Reply:
Here and in all other instances